src/interface/src/app/signup/thank-you/thank-you.component.html

```diff
@@ -0,0 +1,20 @@
+<div class="thankyou-root">
```

Comment:
Looks like you are adding this element just for styling. You can remove this element and instread of styling `thankyou-root` you can do :
```
:host {
  display: flex;
  height: 100%;
}
```